app/Services/PatientService.php

```diff
@@ -74,4 +75,16 @@ public function checkDuration($shift, $duration)
             }
         } 
     }
+
+    public function codeResend($data)
+    {
+        $patient = $this->repository->findPatientByEmail($data);
+        (!Hash::check($data['password'], $patient->password)) ? abort(Response::HTTP_UNAUTHORIZED) : "" ;
+        ($patient->verified_at) ? abort(Response::HTTP_BAD_REQUEST) : "" ;
```

Comment:
Use `full if condition` if you're going to return anything in a variable

Reply:
done